README.md

```diff
@@ -37,46 +41,45 @@ endpoint permissions in the documentation as well you need to name the function
+| **middlewares**							                    | object		      | 	❌	| Configuration parameters for parsing middlewares.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| **middlewares**.formatter					            | function		    | 	❌	| Custom formatter function for middleware. If not used, the default one will be used.                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| **middlewares**.middlewareName			         | string		      | 	✅	| Name of the middleware responsible for handling API permissions.                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| **middlewares**.closure					              | string		      | 	✅	| Name of the permission middleware closure.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| **middlewares**.maxParamDepth					              | number		      | 	❌	| Max copying depth of middleware parameter. By default is 5.                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
@kot-1999 rename `Max copying depth of middleware parameter. By default is 5.` to `Max depth of middleware parameter. Default value is 5.`